### ISSUE TITLE
freeze all the batch norm

### ISSUE THREAD

**Issue by foralliance** [chengyangfu/caffe#24]
In **READEME**:
`Train and Eval the DSSD model. In this script, we try to fine-tune the entire network. In order to sucessfully finetune the network, we need to freeze all the batch norm related layers in Caffe.

How do you understand this? Why freeze all the batch norm related layers in Caffe?
